CHANGELOG.md

```diff
@@ -4,6 +4,72 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.5.0 - 2018-12-02
+
+### Added
+
+- We now push the ingress custom resource defintion and initial
+  configuration, allowing the ingress operator to configure itself
+  without referencing the deprecated `cluster-config-v1` resource.
+
+### Changed
+
+- Pull secret documentation now points to
+  [try.openshift.com](https://try.openshift.com) for pull-secret
+  acquisition, instead of pointing at `account.coreos.com`.  Users
+  should update their pull secrets, as old CoreOS pulls secrets are
+  likely to stop working with future installer releases.  For this
+  release, either `try.openshift.com` or `account.coreos.com` pull
+  secrets will work.
+- If the automatic bootstrap teardown (which landed in 0.4.0) times
+  out waiting for the `bootstrap-complete` event, the installer exits
+  with a non-zero exit code.  We had ignored watcher timeouts in 0.4.0
+  due to concerns about watcher robustness, but the current watcher
+  code has been reliable in our continuous integration testing.
+- The etcd service now uses [selectors][kube-selector] to determine
+  the pods it exposes, and the explict etcd endpoints object is gone
+  (replaced by the one Kubernetes maintains based on the selector).
+- On AWS, both masters and worker have moved from t2.medium nodes
+  m4.large nodes (more on AWS instance types
+  [here][aws-instance-types] to address CPU and memory constraints.
+- On AWS, master volume size has been bumped from 30 GiB to 120 GiB to
+  increase our baseline performance from on [gp2's sliding IOPS
+  scale][aws-ebs-gp2-iops] from the 100 IOPS floor up to 360 IOPS.
+  Volume information is not currently supported by [the cluster-API
```

Comment:
Not sure why this belongs in Changelog

Reply:
> Not sure why this belongs in Changelog

The whole entry?  Just the cluster-API caveat?

Reply:
@abhinavdahiya bump

Reply:
> AWS provider's
  `AWSMachineProviderConfig`][cluster-api-provider-aws-012575c1-AWSMachineProviderConfig],
  so this change is currently limited to masters created by the
  installer.

this part

Reply:
I guess we could leave that off. We don't support scaling masters anyway. @wking what do you think? I could go either way.

Reply:
> We don't support scaling masters anyway.

Doesn't the cluster API bring a master back up if it dies (because someone accidentally deletes it in the AWS console, I dunno)?  That would hit this limitation.

Reply:
Probably, but etcd wouldn't come up. Let's go ahead and leave that note since I think it may prove helpful to our future selves.

Reply:
> Probably, but etcd wouldn't come up.

Ah.  I was hoping coming up with the same hostname would be enough to get back into the cluster ;).  But maybe it needs a secret that would have been lost with the old volume.

Reply:
The etcd client has an internal ID that will be different when the replacement comes up. The cluster will not let that client participate until it has been joined.